guacamole-common/pom.xml

```diff
@@ -26,7 +26,7 @@
     <groupId>org.apache.guacamole</groupId>
     <artifactId>guacamole-common</artifactId>
     <packaging>jar</packaging>
-    <version>1.1.0</version>
+    <version>1.2.0</version>
```

Comment:
Has guacamole-common changed? I'm not seeing any changes from 1.1.0 according to git.

Reply:
Good point - reverted that.